### DISCUSSION TITLE
Disabling Animations and Hotspots in AR Mode on Android

### DISCUSSION THREAD

**Discussion by samsloan** [google/model-viewer#4580] [Q&A] [answered]
Hi there,

I'm using the 3D Model Viewer for an AR project on Android. My model has animations and hotspots, which work well in the standard 3D view. However, in AR mode, I'd like to display only the model without these animations and hotspots, similar to the Quick Look AR experience on iOS.

Is there a straightforward way to disable animations and hotspots when the model is viewed in AR on Android devices?

Cheers,
Sam

**Comment by elalish** [comment, marked as answer]
You can add an `ar-status` event listener and use that to play/pause the animation and set the hotspots to display: none. https://modelviewer.dev/docs/index.html#entrydocs-augmentedreality-events-arStatus
